cmd/host_windows.go

```diff
@@ -35,18 +40,16 @@ that runs inside a VM.
 | vsock-host | <----- AF_VSOCK -----> [ VM ] <----- AF_VSOCK -----> | vsock-peer   |
  ----------------------------------------------------------------------------------`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			ipv6, err := cmd.Flags().GetBool("ipv6")
-			if err != nil {
-				return err
-			}
-			hosts, err := cmd.Flags().GetStringToString("built-in-hosts")
-			if err != nil {
-				return err
-			}
-			upstreamServers, err := cmd.Flags().GetStringArray("upstream-servers")
+			ipv6 := hostViper.GetBool("ipv6")
+			upstreamServers := hostViper.GetStringSlice("upstream-servers")
+
+			allSettings := hostViper.AllSettings()
+			builtInHosts := allSettings["built-in-hosts"]
+			hosts, err := cast.ToStringMapStringE(builtInHosts)
 			if err != nil {
-				return err
+				logrus.Errorf("reading built-in-hosts value: %v", err)
 			}
```

Comment:
```suggestion
			hosts := runViper.GetStringMapString("built-in-hosts")
```
(Again, I might be missing something…)